Draft: patch 113084

ahoblitz opened the pull request:
Fixes #113084


cc @svekars @carljparker

Commit:
patch 113084

ahoblitz commented:
@pytorchbot label "topic: not user facing"

Commit:
fix some spacing

kit1980 commented on torch/_refs/__init__.py:
```diff
@@ -1103,38 +1103,123 @@ def bitwise_left_shift(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
+
+    Example:
+        >>> a = torch.tensor([1.0, -2.0, 3.0])
+        >>> b = torch.tensor([-1.0, 2.0, -3.0])
```
`b` should be probably different from the result to properly show what the function does.

Commit:
update example based on feedback

kit1980 reviewed (changes requested):
@ahoblitz Did you use help of ChatGPT or something like that? There are a lot of details I'm not sure are correct.

Also all the examples are wrong, should not use numpy arrays.

kit1980 commented:
@ahoblitz please verify all details you provide are correct and also that all the examples are correct.

Commit:
np.array => torch.tensor

ahoblitz commented:
@kit1980 The values were computed by hand but they should be correct. Updated to use `torch.tensor` since we are in torch rather than numpy

ahoblitz reviewed (commented):
Updated to use torch.tensor

kit1980 commented:
@ahoblitz I still see incorrect details and verifying some other takes time.
Please verify/remove the details.

Commit:
simplify comments and adjust formatting

ahoblitz commented:
@kit1980 
I have
- [X] Simplified some comments
- [X] Fixed some details with the linter
- [X] Validated inputs/outputs match execution (in addition to mental math), no changes seem to be required here

I am not sure if this resolves the details you would like verified or removed. One other potential subjective issue I saw may be around type aliasing and using `TensorLike` for example inputs over `tensor`, but please let me know if you see this change or any other specific changes as appropriate

kit1980 commented:
@ahoblitz For `copysign`, can left argument be a scalar?

kit1980 commented:
Sorry, we've realized that the docstrings already exist, just defined in a different file: https://github.com/pytorch/pytorch/blob/main/torch/_torch_docs.py#L3071

ahoblitz commented:
I will close this since due to the docstrings existing elsewhere
